docs/reference/security/rbac.md

```diff
@@ -65,28 +65,33 @@ This helps us compose policies together. For example, we could attach a very per
 
 ## Resource naming - ARNs
 
-lakeFS uses [ARN identifier](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_identifiers.html#identifiers-arns){:target="_blank"} - very similar in structure to those used by AWS.
-The resource segment of the ARN supports wildcards: use `*` to match 0 or more characters, or `?` to match exactly one character.
+lakeFS uses [ARN identifier](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_identifiers.html#identifiers-arns){:target="_blank"} - very similar in structure to those used by AWS. 
+For example:
 
-Additionally, the current user's ID is interpolated in runtime into the ARN using the `${user}` placeholder.
+```text
+arn:lakefs:auth:::user/jane.doe
+arn:lakefs:fs:::repository/myrepo/object/foo/bar/baz
+```
 
-Here are a few examples of valid ARNs within lakeFS:
+The resource segment of the ARN supports wildcards: use `*` to match 0 or more characters, or `?` to match exactly one character. For example,
+you can specify `user/*` to mean all users or `repository/*` to mean all paths under a repository, as in the following examples:  
 
 ```text
-arn:lakefs:auth:::user/jane.doe
 arn:lakefs:auth:::user/*
+arn:lakefs:fs:::repository/*
+```
+
+Here are a few more examples of valid ARNs within lakeFS:
```

Comment:
Thanks @arielshaqed , that makes sense.  I changed the format of the example into a table so that their explanations will be readable. Does this look better or a table for examples is confusing? 
(p.s. Apologies for the delay..!)